src/app/core/entity/entity-remove.service.ts

```diff
@@ -134,9 +126,10 @@ export class EntityRemoveService {
     ).subscribe(async (next) => {
       if (next) {
```

Comment:
I guess this is a leftover but I don't really see a point of combining these two observables. We do something `onAction` and we do nothing `afterDismissed` so we can just leave the second part out.
I am happy to refactor this if wanted.